packages/block-editor/src/components/url-input/stories/index.story.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import URLInput from '..';
+
+export default {
+	component: URLInput,
+	title: 'BlockEditor/URLInput',
+};
+
+export const Default = {
+	render: function Template( props ) {
+		const [ value, setValue ] = useState();
+		return <URLInput onChange={ setValue } value={ value } { ...props } />;
+	},
+	args: {},
```

Comment:
The story seems a bit useless without any controls or any included potential search results.

![Screenshot 2024-08-06 at 14 32 25](https://github.com/user-attachments/assets/f34d5e58-ce50-406c-adae-517e98df3c55)

Reply:
You're right 😂 I will remove it.